apps/mobile/src/app/(tabs)/_layout.tsx

```diff
@@ -29,7 +29,7 @@ export default function TabLayout() {
             header: AssetsNavbar,
             title: 'Home',
             tabBarButtonTestID: 'home-tab',
-            tabBarButton: ({ children, ...rest }) => {
+            tabBarButton: ({ children, ref, ...rest }) => {
```

Comment:
If the variable isn't used, we prefix the name with an underscore, e.g. _ref, in the web app. Curious why lint didn't complain here.

Reply:
good point. It complained when I didn't have the ref as the type of the ref is slightly different. 🤷